browse/static/css/arXiv.css

```diff
@@ -664,28 +664,47 @@ footer .sorry-app-links .a11y-main-link {
   justify-content:center;
 }
 .banner-minimal a {
-  -webkit-box-shadow: 0px 3px 6px 0px #000000);
+  -webkit-box-shadow: 0px 3px 6px 0px #000000;
```

Comment:
What a great change!!! Why was there a `)` in the beginning?

Reply:
looks like a typo. I think not everyone who works on the CSS uses code editors that will aggressively highlight syntax errors